apps/css-workshop/src/components/Anchor.astro

```diff
@@ -0,0 +1,18 @@
+---
+type Props = {
+  external?: 'true';
+  status?: 'informational' | 'positive' | 'warning' | 'negative';
+  underline?: 'true';
```

Comment:
This should be a boolean prop

```ts
underline?: boolean
```

implemented like this:

```jsx
data-iui-underline={underline ? 'true' : undefined}
```

and used like this:
```jsx
<Anchor_ underline>
```

Reply:
Same with `external`, which should be a boolean and should also be renamed to `isExternal` to match react.

Reply:
Both changes made.